examples/mnist/Instructions.md

```diff
@@ -0,0 +1,27 @@
+<p align="center"><b>Setup, build and run tiny-dnn on various operating systems</b><p align="center">
+
+<b>tiny-dnn</b> is a C++11 implementation of deep learning. It is suitable for deep learning on limited computational resource, embedded systems and IoT devices.
+
+
```

Comment:
@karandeepdps Thank you for the changes. It looks good to me.
https://github.com/karandeepdps/tiny-dnn/blob/75639d9c034e0c4ccac862b59c35e9d0823dd8aa/examples/mnist/Instructions.md

@edgarriba Please check the document again.